### DISCUSSION TITLE
Genymotion plugin in Android Studio doesn't show virtual devices created when hypervisor is set to QEMU.

### DISCUSSION THREAD

**Discussion by DangXuanThong** [Genymobile/feedback#52] [Genymotion Desktop] [answered]
I set genymotion hypervisor to QEMU and created an emulator in it. But the plugin in Android Studio does not show it, only shows the emulators created when in virtual box hypervisor.

**Comment by agavignet** [comment]
Hi @DangXuanThong,

Indeed we did not update our plugin to support QEMU yet. The team is currently planning it.
Can't give you an ETA yet but it's in our pipe :)

Adrien

**Comment by agavignet** [comment, marked as answer]
Hi @DangXuanThong 
A new version has been released with the QEMU support. You can check the release note [here](https://www.genymotion.com/blog/release-note/genymotion-plugins-2-0-0/).
Thanks for your patience
